avocado/utils/cpu.py

```diff
@@ -152,7 +172,37 @@ def get_cpu_arch():
     return platform.machine()
 
 
-def cpu_online_list():
+def get_family():
```

Comment:
@sathnaga would you please include selftests for `get_family` as it's a new interface?

Reply:
sure

Reply:
sure